### DISCUSSION TITLE
Found error log when click edit proxy host

### DISCUSSION THREAD

**Discussion by rokoman** [ZoeyVid/NPMplus#480] [Q&A] [answered]
![e1](https://github.com/ZoeyVid/nginx-proxy-manager/assets/1106739/fb877690-c218-46da-a854-104a17d111e8)

[Access   ] › ✖  error     access_lists:list undefined validation failed

**Comment by Zoey2936** [comment]
can I see the config of the access list?

**Comment by rokoman** [comment]
> can I see the config of the access list?

I don't config access list. but i test on other user no view access list.

**Comment by Zoey2936** [comment]
I don't understand

**Comment by rokoman** [comment]
![user1](https://github.com/ZoeyVid/nginx-proxy-manager/assets/1106739/a894443e-d810-4c36-957a-d64bd39bb6b3)

I add new user and set permission hidden access list, when edit proxy host i found error log (access_lists:list undefined validation failed).

On admin user no error log when edit proxy host.

**Comment by Zoey2936** [comment, marked as answer]
I can confirm it, but beside the log everything works

**Comment by Zoey2936** [comment]
so I would close it, since it is only a misleading log and no issue from my view
